NorthstarDLL/squirrel/squirrelautobind.h

```diff
@@ -21,16 +21,16 @@ class __squirrelautobind;
 		__squirrelautobind CONCAT2(__squirrelautobind, __LINE__)(                                                                          \
 			[]()                                                                                                                           \
 			{                                                                                                                              \
-				if constexpr (runOnContext & ScriptContext::UI)                                                                            \
+				if constexpr ((runOnContext) & ScriptContext::UI)                                                                            \
```

Comment:
Ok, this is hitting the limits of my C++ knowledge. Why does adding (seemingly unnecessary) brackets fix the issue? ^^"

Reply:
because of how macros work

runOnContext is replaced witb an expression, for example:
`ScriptContext::SERVER | ScriptContext::CLIENT`.

`ScriptContext::SERVER | ScriptContext::CLIENT & ScriptContext::UI` causes a compile error because it doesn't know how to combine it.